src/components/wallet/methods/trustAsset.ts

```diff
@@ -45,27 +45,29 @@ async function getAssetAndIssuer(wallet: Wallet) {
     homeDomain[homeDomain.length - 1] !== '/'
       ? homeDomain
       : homeDomain.slice(0, -1)
+  let homeDomainURL
   try {
-    new URL(homeDomain)
+    homeDomainURL = new URL(homeDomain)
   } catch (e) {
     throw 'anchor home domain is not a valid URL using HTTPS'
   }
 
   // if the issuer was not provided, extract if from the home domain's TOML
   if (!issuer && homeDomain) {
-    let resp = await fetch(homeDomain + '/.well-known/stellar.toml')
-    let tomlContents = parse(await resp.text())
-    if (!tomlContents.CURRENCIES) {
+    let toml = await StellarTomlResolver.resolve(homeDomainURL.origin)
```

Comment:
This fails when attempting to add an asset via a home domain.

### Flow to reproduce
1. add asset
2. Asset code = `SRT` or `MXN`
3. Anchor home domain = `https://testanchor.stellar.org` or `https://www.dev.anchormxn.com`

### Errors received  

```
GET https://https//www.dev.anchormxn.com/.well-known/stellar.toml net::ERR_NAME_NOT_RESOLVED

Uncaught (in promise) Error: Network Error
    at createError (stellar-wallet.entry.js:formatted:36437)
    at XMLHttpRequest.handleError (stellar-wallet.entry.js:formatted:36747)

Error in trust transaction
{
config: 12 values
{
url: https://https://www.dev.anchormxn.com/.well-known/stellar.toml
method: get
headers: 1 values
{
Accept: application/json, text/plain, */*
}
transformRequest: 1 values
{
0: Function
}
transformResponse: 1 values
{
0: Function
}
timeout: 0
adapter: Function
xsrfCookieName: XSRF-TOKEN
xsrfHeaderName: X-XSRF-TOKEN
maxContentLength: 102400
validateStatus: Function
data: undefined
}
request: 0 values
{
}
response: undefined
isAxiosError: true
toJSON: Function
}
```